crates/router/src/connector/stripe/transformers.rs

```diff
@@ -124,6 +120,24 @@ pub struct PaymentIntentRequest {
     pub payment_method_types: Option<StripePaymentMethodType>,
 }
 
+// Field rename is required only in case of serialization as it is passed in the request to the connector.
+// Deserialization is happening only in case of webhooks, where fields name should be used as defined in the struct.
+// Whenever adding new fields, Please ensure it doesn't break the webhook flow
+#[derive(Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct ConnectorMetaData {
+    // merchant_reference_id
+    #[serde(rename(serialize = "metadata[order_id]"))]
+    pub order_id: String,
+    #[serde(rename(serialize = "metadata[txn_id]"))]
+    pub txn_id: String,
```

Comment:
Do we need to send txn_id and txn_uuid?

Reply:
there doesn't seem to be any significant use of it, since it was present before my change so just refactored the variable name.